website/views/shared/modals/invite-friends.jade

```diff
@@ -54,6 +54,10 @@ script(type='text/ng-template', id='modals/invite-party.html')
     div(ng-if='!group._id')
       h4(style='margin-top:0')=env.t('nameYourParty')
       input.form-control#new-group-name.input-medium.option-content(required, type='text', ng-model='group.name')
+    p.alert.alert-warning(ng-if="getMemberAndInviteCount(group) > 25") 
+      | You are approaching the 30 member party cap!
```

Comment:
to split the translation string in multiple lines you can use the `\n` character (for example `    "unorderedListMarkdown": "+ First item\n+ Second item\n+ Third item"
`). The `{{getMemberAndInviteCount(group)}}` code instead can be passed as a variable like `              button.btn.btn-primary.btn-block(ng-show='selectedEgg', ng-click='sellInventory()')=env.t('sellForGold', {itemType: "{{selectedEgg.text()}}", gold: "{{selectedEgg.value}}"})
`